libs/tup-components/src/projects/ProjectsTable.tsx

```diff
@@ -34,53 +32,47 @@ export const ProjectsTable: React.FC = () => {
       columns,
       data: data ?? [],
     });
-
   if (isLoading) {
     return <LoadingSpinner />;
   }
-
   if (error) {
     return <Message type="warn">Unable to retrieve projects.</Message>;
   }
   return (
-    <table
-      {...getTableProps()}
-      className={`multi-system InfiniteScrollTable o-fixed-header-table ${[
-        'root',
-      ]}`}
-    >
-      <thead>
-        {headerGroups.map((headerGroup) => (
-          <tr
-            {...headerGroup.getHeaderGroupProps()}
-            className={styles['header']}
-          >
-            {headerGroup.headers.map((column) => (
-              <th {...column.getHeaderProps()}>{column.render('Header')}</th>
-            ))}
-          </tr>
-        ))}
-      </thead>
-      <tbody {...getTableBodyProps()} className={styles['rows']}>
-        {rows.length ? (
-          rows.map((row, idx) => {
-            prepareRow(row);
-            return (
-              <tr {...row.getRowProps()}>
-                {row.cells.map((cell) => (
-                  <td {...cell.getCellProps()}>{cell.render('Cell')}</td>
-                ))}
-              </tr>
-            );
-          })
-        ) : (
-          <tr>
-            <td colSpan={5}>No active projects found.</td>
-          </tr>
-        )}
-      </tbody>
-    </table>
+    <div className="o-fixed-header-table">
+      <table {...getTableProps()}>
+        <thead>
+          {headerGroups.map((headerGroup) => (
+            <tr
+              {...headerGroup.getHeaderGroupProps()}
+              className={styles['header']}
```

Comment:
The `className={styles['header']}` can be removed, because `<table>`[^1]s has global styles (after #70).

Actually, all of `libs/tup-components/src/projects/Projects.module.css` can be deleted.

[^1]: And `<thead>`, `<tbody>`, `<tr>`, `<th>`, `<td>`, et cetera will be styled by global styles.